src/FileOnQ.Imaging.Raw/Processors/DcrawProcessor.cs

```diff
@@ -14,15 +14,31 @@ namespace FileOnQ.Imaging.Raw
 	public class DcrawProcessor : IImageProcessor
 	{
 		/// <inheritdoc cref="IImageProcessor"/>
-		public virtual bool Process(RawImageData data)
+		public virtual unsafe bool Process(RawImageData data)
 		{
+			var parameters = LibRaw.GetOutputParameters(data.LibRawData);
+			SetOutputParameters(parameters);
+
 			var errorCode = LibRaw.DcrawProcess(data.LibRawData);
 			if (errorCode != LibRaw.Error.Success)
 				throw new RawImageException<LibRaw.Error>(errorCode);
 
 			return true;
 		}
 
+		/// <summary>
+		/// Configures post processing LibRaw output parameters. Any
+		/// value set on <see cref="LibRaw.DcrawPostProcessingParameters"/>
+		/// will be applied, no need to perform additional save operation.
+		/// </summary>
+		/// <param name="parameters">
+		/// The current output parameters.
+		/// </param>
+		protected virtual unsafe void SetOutputParameters(LibRaw.DcrawPostProcessingParameters* parameters)
```

Comment:
@mitchelsellers I left this as a pointer to keep the integration simple. If we want to remove pointers from downstream integrations we will have to copy the data back to native memory which will be cumbersome. For example we could update this to pass a standard c# struct instead of a pointer. Once this method completes, we will need to update each property on the native memory via the original pointer.